client/src/modules/dashboard/Events/graphql/queries.ts

```diff
@@ -129,14 +129,112 @@ export const updateEvent = gql`
       url
       streaming_url
       capacity
+      start_at
+      ends_at
+      image_url
+      invite_only
       tags {
         id
         name
       }
+      sponsors {
+        sponsor {
+          name
+          website
+          logo_path
+          type
+          id
+        }
+      }
+      venue {
+        id
+        name
+        street_address
+        city
+        postal_code
+        region
+        country
+      }
+      chapter {
+        id
+        name
+      }
     }
   }
 `;
 
+export const EVENT_WITH_CHAPTER_FRAGMENT_UPDATE = gql`
+  fragment updateFragmentEventWithChapter on EventWithChapter {
+    id
+    name
+    description
+    url
+    streaming_url
+    capacity
+    start_at
+    ends_at
+    capacity
+    image_url
+    invite_only
+    chapter {
+      id
+      name
+    }
+    tags {
+      id
+      name
+    }
+  }
+`;
+
+export const EVENT_WITH_EVERYTHING_FRAGMENT_UPDATE = gql`
+  fragment updateFragmengEventWithEverything on EventWithEverything {
+    id
+    name
+    description
+    url
+    streaming_url
+    capacity
+    start_at
+    ends_at
+    capacity
+    image_url
+    invite_only
+    chapter {
+      id
+      name
+    }
+    tags {
+      id
+      name
+    }
```

Comment:
```suggestion
  fragment updateFragmentEventWithEverything on EventWithEverything {
    ...updateFragmentEventWithChapter
```
We can DRY this a fair bit.